### ISSUE TITLE
Update ReadTheDocs to v2

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#1144]
Just logged in to ReadTheDocs to have a look at https://earth.bsc.es/gitlab/es/autosubmit/-/merge_requests/380 when I noticed this warning in ReadTheDocs:

![image](https://earth.bsc.es/gitlab/es/autosubmit/uploads/03c9980fec7492962b638cfca8983f13/image.png)

Link: https://blog.readthedocs.com/migrate-configuration-v2/

>We plan to start failing builds not using configuration file version 2 on September 25, 2023.

So I guess that whatever was supposed to happen is already happening? Not sure if the message is in place for all users or if our configuration needs to be updated. Will take a look later and close this if no action is required from our side.

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Oct 5, 2023, 14:45

Hello, 

But we are already using version 2 , no? 

https://earth.bsc.es/gitlab/es/autosubmit/-/blob/master/.readthedocs.yaml

**Comment by kinow** [comment]
Yup, not sure why the message is happening. I will search around and confirm we don't have to do anything and close it if that's just a general warning to all users.

**Comment by dbeltrankyl** [comment]
I believe this is done. If not, re-open it
